### DISCUSSION TITLE
Error when opening Fabric 17.1

### DISCUSSION THREAD

**Discussion by Hydragon0107** [FabricMC/fabric-api#1776] [Player support]
So I wanted to use world edit on fabric in a 17.1 world so I reinstalled fabric on 17.1 and removed the all the 16.5 mods (one of which was the origins mod) and the 16.5 API and replaced it with the world edit file and 17.5 fabric API but when I started Minecraft with fabric 1.17 I got the message: Could not find required mod: origins requires {minecraft @ [1.16.x]} . I tried some other stuff like changing the API version and what version of fabric I was using but I didn't do anything, I checked the .minecraft folder to see if the origins mod was still in there but I couldn't find anything. I'm probably making a dumb mistake but I'm not sure how to fix it.

**Comment by kanpov** [comment]
@Hydragon0107 Your installation of the Origins mod only works with Minecraft 1.16.

Please, update to the newest 1.17 version!

**Comment by AlexdaCat** [comment]
> origins requires {minecraft @ [1.16.x]}

you need to install the latest Origins version
